examples/Huggingface_Transformers/Image_classification_docker/README.md

```diff
@@ -0,0 +1,28 @@
+# HF-torchserve-pipeline
+ 
+This repository contains an example to deploy models with third-party dependencies (like 🤗 Transformers, sparseml etc) on Torchserve servers as ready-for-usage Docker containers on cloud services like AWS.  
+
+For the context of this repository, we would deploy the models on an AWS [`t2.micro`](https://aws.amazon.com/ec2/instance-types/) instance which can be used for free (for 750 hours) on a new AWS account. We work with a 🤗 MobileViT Transformer [model](https://huggingface.co/apple/mobilevit-xx-small) for the task of image classification by using its [`pipeline`](https://huggingface.co/docs/transformers/main_classes/pipelines) feature, the handler code in `scripts` can also be used as a simplistic template to deploy an 🤗 `pipeline`.
+
+This work can also be [extended to deploy *any* 🤗 `pipeline` for *any* supported task with Torchserve](https://github.com/tripathiarpan20/HF-torchserve-pipeline/tree/main/HF-only#instructions-to-use-any--model-from-the-hub-for-any-task-supported-by-the--pipeline).
+
+This work *may* also be extended to deploy the Torchserve Docker containers with HF models at scale with [AWS Cloudformation](https://github.com/pytorch/serve/tree/master/examples/cloudformation) & [AWS EKS](https://github.com/pytorch/serve/tree/master/kubernetes/EKS) as explained in the official Torchserve repo & [AWS Sagemaker](https://github.com/tescal2/TorchServeOnAWS/tree/master/3_torchserve_byoc_with_amazon_sagemaker), incorporating utilities like AWS ELB & Cloudwatch.
+
+We would also benchmark the REST API calls in time units and compare the model performances for the following approaches: 
+* Deploying the [MobileViT XX Small](https://huggingface.co/apple/mobilevit-xx-small) Huggingface model with a custom torchserve handler. (refer `HF-only` directory)
+* Deploying the [MobileViT XX Small](https://huggingface.co/apple/mobilevit-xx-small) Huggingface model in [scripted mode](https://pytorch.org/tutorials/beginner/Intro_to_TorchScript_tutorial.html) with a custom torchserve handler. (refer `HF-scripted` directory)
+
+## Latest version
+The latest changes are reflected in [this repo](https://github.com/tripathiarpan20/HF-torchserve-pipeline)
```

Comment:
I'd recommend integrating your work natively in torchserve since that will provide the most convenience to end user and we can credit you in the release notes and also in any blog we end up publishing together. It's tricky for me to accept this line because then it's dependent on you continuing to maintain your repo but if you contribute it directly to serve then you can do as many followup PRs as you like and if for whatever reason you can't continue someone else can pick up the work

Reply:
That makes sense, it would maintain the quality of the Torchserve repo indeed and allow future contributions. Would remove these lines too.